.travis.yml

```diff
@@ -10,26 +10,26 @@ matrix:
       dist: xenial
       sudo: required
       compiler: gcc
-      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON -DEXIV2_ENABLE_CURL=ON -DCMAKE_CXX_STANDARD=98"
+      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON -DEXIV2_ENABLE_CURL=ON cmake .. -DCMAKE_CXX_STANDARD=11 -DCMAKE_CXX_FLAGS=-Wno-deprecated"
```

Comment:
I see 2 problems here:
- First of all `CMAKE_OPTIONS` is a variable with cmake options that will be passed later to CMake. The addition `cmake ..` does not make sense here.
- I do not understand the change `CMAKE_CXX_STANDARD=11`. We are still requiring to use C++98 in `0.27-maintenance` . The usage of C++11 is not possible since we are using the `auto_ptr` stuff. Apart from that, we are not using any c++11 stuff in this branch, so we would not gain anything by telling to the compiler to use the C++11 standard.

Reply:
You are quite right about C++11.  The puzzle to solve is the CI overflowing with deprecation warnings and then saying "log too long" and marks the job in error.